tf2_ros/include/tf2_ros/transform_listener.h

```diff
@@ -41,6 +41,10 @@
 #include "tf2_ros/buffer.h"
 #include "tf2_ros/visibility_control.h"
 
+namespace
+{
+using thread_ptr = std::unique_ptr<std::thread, std::function<void(std::thread *)>>;
```

Comment:
my personal style would just be to put the `using` just above the first (and only) declaration of this type in line 115.

Reply:
Since it's only used once that is a lot easier to read.